tests/director/tests/application_templates_api_test.go

```diff
@@ -832,6 +832,56 @@ func TestDeleteApplicationTemplate(t *testing.T) {
 	saveExample(t, deleteApplicationTemplateRequest.Query(), "delete application template")
 }
 
+func TestDeleteApplicationTemplateWithCertSubjMapping(t *testing.T) {
```

Comment:
I think it will be better if the test creates 2 cert subject mappings and at the end checks that both of them are deleted. That way we will check the mass deletion.